packages/hint/docs/contributor-guide/getting-started/events.md

```diff
@@ -3,7 +3,8 @@
 Information is shared internally via `event`s. `connector`s and `parser`s can
 create them, while `parser`s and `hint`s consume them.
 The following is a list of all the events common to all `connector`s, with
-their signature, and the `interface` they implement.
+their signature, and the `interface` they implement. The exception is the
+`local connector` that needs the `HTML parser` to emit these events:
```

Comment:
FYI we have some adjacent edits in my typed events PR:
https://github.com/webhintio/hint/pull/1397/files#diff-a9286171f2cf5bcdf385128b16fa25ad